src/Components/test/E2ETest/Infrastructure/ServerFixtures/ServerFixture.cs

```diff
@@ -37,7 +37,9 @@ protected static string FindSolutionDir()
         private static Dictionary<string, string> FindProjects()
         {
             var solutionDir = FindSolutionDir();
-            return Directory.GetFiles(solutionDir, "*.csproj", SearchOption.AllDirectories)
+            return 
+                Directory.GetFiles(solutionDir, "*.csproj", SearchOption.AllDirectories)
```

Comment:
I had a go at fixing this in my PR. We really don't need to scan the entire sln directory especially given that it has `node_modules` sitting in some of them:

```C#
var testAssetsDirectories = new[]
{
    Path.Combine(solutionDir, "test", "testassets"),
    Path.Combine(solutionDir, "blazor", "testassets"),
};

return testAssetsDirectories.SelectMany(d => new DirectoryInfo(d).EnumerateDirectories())
    .ToDictionary(d => d.Name, d => d.FullName);
```

Reply:
Could we just do something like https://github.com/aspnet/AspNetCore/blob/master/src/Mvc/Mvc.Testing/src/Microsoft.AspNetCore.Mvc.Testing.targets and just read the assembly attributes?

It's more resilient and there's no guessing involved

Reply:
Feel free to file a work item for that, I'm not going to go on an adventure as part of this unrelated change.

Reply:
Fair enough

Reply:
I'm going to do Pranav's solution to this for now.